sqliter-driver/src/nativeCommonTest/kotlin/co/touchlab/sqliter/DatabaseConnectionTest.kt

```diff
@@ -276,20 +265,23 @@ class DatabaseConnectionTest {
                 bindString(2, "asdf")
                 executeInsert()
             }
+            it.withStatement("insert into test(num, str)values(?,?)") {
+                bindLong(1, 233)
+                bindString(2, "asdfg")
+                executeInsert()
+            }
         }
 
-        assertEquals(1, conn2.longForQuery("select count(*) from test"))
+        assertEquals(1, conn1.longForQuery("select count(*) from test"))
```

Comment:
Fixed this assertion to demonstrate that `conn2` did nothing to `conn1` database (I assumed it was a copy/paste mistake)